client/src/app/routes/nova/OutputPage.tsx

```diff
@@ -47,6 +50,20 @@ const OutputPage: React.FC<RouteComponentProps<OutputPageProps>> = ({
 
     const { blockId, transactionId, outputIndex, isSpent, transactionIdSpent } = outputMetadataResponse ?? {};
 
+    // @ts-expect-error TODO: Remove this ignore once included/spent are honoured in the type https://github.com/iotaledger/iota-sdk/issues/1884
```

Comment:
this issue got resolved so we could remove these comments: https://github.com/iotaledger/iota-sdk/issues/1884